desktop_server/src/com/mygdx/game/server/ecs/entityfactory/EntityFactory.java

```diff
@@ -0,0 +1,22 @@
+package com.mygdx.game.server.ecs.entityfactory;
```

Comment:
Wydaje mi się, że ta fabryka jest tu i w cliencie, nie można dać do core?

Reply:
Nie, później się to zmienia i mają osobne implementacje.